src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -97,9 +97,15 @@ public function process(ContainerBuilder $container)
 
             // We must assume that the class value has been correctly filled, even if the service is created by a factory
             $class = $container->getParameterBag()->resolveValue($def->getClass());
-
             $interface = 'Symfony\Component\EventDispatcher\EventSubscriberInterface';
+
             if (!is_subclass_of($class, $interface)) {
+                if (!class_exists($class, true)) {
+                    throw new \RuntimeException(
```

Comment:
This is just a special subcase of the outer `if`, I see no reason to introduce a new semantic here. Btw, you can't tell here why the class can't be loaded. Could be a typo in the class name, in the file name, or something else.